flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/CDU/A320_Neo_CDU_PerformancePage.js

```diff
@@ -1071,7 +1077,7 @@ class CDUPerformancePage {
         mcdu.engineOutAccelerationAltitudeIsPilotEntered = false;
         SimVar.SetSimVarValue("L:A32NX_ENG_OUT_ACC_ALT", "feet", alt);
     }
-    static UpdateThrRedAccFromDestination(mcdu) {
+    static async UpdateThrRedAccFromDestination(mcdu) {
         const destination = mcdu.flightPlanManager.getDestination();
         const elevation = destination ? destination.altitudeinFP : SimVar.GetSimVarValue("GROUND ALTITUDE", "feet");
```

Comment:
```suggestion
        let elevation = SimVar.GetSimVarValue("GROUND ALTITUDE", "feet");
        if (destination) {
            elevation = await mcdu.facilityLoader.GetAirportFieldElevation(destination.icao);
        }
```

Reply:
All destination GA reverted back to original code. The GA (destination) will be implemented when the secondary flight plan and GA track are complete.